client/tx/factory.go

```diff
@@ -76,8 +76,12 @@ func NewFactoryCLI(clientCtx client.Context, flagSet *pflag.FlagSet) Factory {
 	feesStr, _ := flagSet.GetString(flags.FlagFees)
 	f = f.WithFees(feesStr)
 
-	gasPricesStr, _ := flagSet.GetString(flags.FlagGasPrices)
-	f = f.WithGasPrices(gasPricesStr)
+	// I.G 24/01/23 - our default for gas-prices essentially disables manually specifying fees in the CLI.
+	// This changes behaviour for fees to override the gas prices
+	if feesStr != "" {
```

Comment:
Shouldn't this be `==`?

Reply:
Good catch. No idea how it worked till now